cloudinstall/placement/ui/__init__.py

```diff
@@ -165,9 +165,6 @@ def build_widgets(self):
         self.deploy_ok_msg = ("\u2713 All the required OpenStack services are "
                               "placed on a machine, and you can now deploy.")
 
-        # Save a new placements configuration once all required services are
-        # placed.
-        self.placement_controller.save()
```

Comment:
Ok cool. I will deal with it later if I need to. Thanks!

On Friday, April 3, 2015, Adam Stokes <notifications@github.com> wrote:

> In cloudinstall/placement/ui/__init__.py
> <https://github.com/Ubuntu-Solutions-Engineering/openstack-installer/pull/535#discussion_r27742254>
> :
>
> > @@ -165,9 +165,6 @@ def build_widgets(self):
> >          self.deploy_ok_msg = ("\u2713 All the required OpenStack services are "
> >                                "placed on a machine, and you can now deploy.")
> >
> > -        # Save a new placements configuration once all required services are
> > -        # placed.
> > -        self.placement_controller.save()
>
> Crap it didn't take my change there, I actually commented that out because
> we weren't passing in the yaml file object. This actually didn't exist in
> the prior implementation of the placements file so I'm going to remove this
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Ubuntu-Solutions-Engineering/openstack-installer/pull/535/files#r27742254>
> .
>